storybook/config.js

```diff
@@ -1,12 +1,14 @@
 import React from 'react'
-import { configure, addDecorator } from '@storybook/react'
+import { configure, addDecorator, addParameters } from '@storybook/react'
 import { withA11y } from '@storybook/addon-a11y'
+import { withInfo } from '@storybook/addon-info'
+import { action } from '@storybook/addon-actions'
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Imports should be sorted alphabetically.](https://app.codacy.com/manual/PrideInLondon/pride-london-web/pullRequest?prid=4765408)